triaging.rst

```diff
@@ -37,6 +36,16 @@ Responsibilities include:
     - Good first issue
     - Other categorizations
 
+Triagers may have some understanding of when a PR should be closed, but the
+final decision must be made by a core developer. This is because the creation
+of a PR constitutes a significant amount of effort from the author. Closing one
```

Comment:
By triager and core developer working together, the author receives careful consideration of the PR whether the PR is accepted or closed which encourages future contributions.

Reply:
Great idea! I think this will provide a significantly more encouraging message. 

Minor phrasing improvement suggestion:

> By having triagers and core developers work together, the author receives a careful consideration of their PR. This encourages future contributors, regardless of whether their PR is accepted or closed.